dask_ml/wrappers.py

```diff
@@ -275,20 +277,38 @@ def fit(self, X, y=None):
             setattr(self, k, v)
         return self
 
-    def partial_fit(self, X, y=None):
+    def __repr__(self):
+        # Have to override, else all the parameters of estimator
+        # are duplicated
+        estimator = repr(self.estimator)
+        class_name = self.__class__.__name__
+        return '{}({})'.format(class_name, estimator)
+
+    def get_params(self, deep=True):
+        out = self.estimator.get_params(deep=deep)
+        out['estimator'] = self.estimator
```

Comment:
Awkward API: what if the estimator parameter has an `estimator` parameter? Raise an error?